elements/nvidia-cuda/README.rst

```diff
@@ -5,8 +5,12 @@ Forked from CSC Finland's repo - https://github.com/CSCfi/diskimage-builder/tree
 
 Install NVidia CUDA packages.
 
-* ``DISTRO_NAME`` The name of the Linux distribution to base on.
-  Recognised settings are ``centos``, ``centos7``, ``rhel7`` and ``ubuntu16``
+* ``DIB_NVIDIA_CUDA_REPO`` Path to the CUDA RPM/Deb repo.
+  Default is ``https://developer.download.nvidia.com/compute/cuda/repos/$distro/x86_64``
+
+* ``DIB_NVIDIA_CUDA_VERSION`` Version of CUDA to install
+  Possible values include: ``7.0-28``, ``7.5-18``, ``8.0.44-1``, ``8.0.61-1``, ``9.0.176-1``.
```

Comment:
This seems likely to go stale.

Reply:
Right, but I think some values are better than no values?

Reply:
I think I read it as 'possible values are...' first time around. This is fine.